automates/text_reading/src/main/scala/org/clulab/aske/automates/OdinActions.scala

```diff
@@ -1001,6 +1232,7 @@ class OdinActions(val taxonomy: Taxonomy, expansionHandler: Option[ExpansionHand
 
   def looksLikeADescr(mentions: Seq[Mention], state: State): Seq[Mention] = {
     val valid = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ "
+    val singleCapitalWord = """^[A-Z]+$""".r
```

Comment:
@alicekwak do you happen to remember what the example for this? Is this used? *line 1246 is commented out, so I am not sure.

Reply:
Oh, it shouldn't have been deleted. I just re-activated the line. Thanks for catching that! It is used to filter out cases like APSGRAPH from descriptions. (i.e. APSGRAPH, APSIM)